lib/aerospike/command/command.rb

```diff
@@ -509,6 +510,8 @@ def execute
             # situation. We will not put back the connection in the buffer.
             @conn.close if @conn
             raise e
+          rescue
```

Comment:
It seems like in this block we always want to close the connection. WDYT about using the `ensure` block instead?

Reply:
@ioquatix ping ^^ and I agree

Reply:
Sorry, I think my code is wrong, we don't want to close the connection unless `parse_result` failed.